builtin/v8/manifest/manifest.go

```diff
@@ -0,0 +1,111 @@
+package manifest
+
+import (
+	"context"
+	"fmt"
+	"io"
+
+	"github.com/filecoin-project/go-state-types/builtin/v8/util/adt"
+
+	"github.com/ipfs/go-cid"
+
+	cbg "github.com/whyrusleeping/cbor-gen"
+)
+
+type Manifest struct {
```

Comment:
This has no documentation so I have no idea what it's for (probably already an issue before you moved it).